core/src/components/checkbox/test/indeterminate/checkbox.e2e.ts

```diff
@@ -2,11 +2,12 @@ import { expect } from '@playwright/test';
 import { test } from '@utils/test/playwright';
 
 test.describe('checkbox: indeterminate', () => {
-  test('should not have visual regressions', async ({ page }) => {
-    await page.goto(`/src/components/checkbox/test/indeterminate`);
+  test('should not have visual regressions', async ({ page, skip }) => {
+    skip.rtl();
 
-    await page.setIonViewport();
+    await page.goto(`/src/components/checkbox/test/indeterminate`);
 
-    expect(await page.screenshot()).toMatchSnapshot(`checkbox-indeterminate-${page.getSnapshotSettings()}.png`);
+    const content = page.locator('#checkboxes');
+    expect(await content.screenshot()).toMatchSnapshot(`checkbox-indeterminate-${page.getSnapshotSettings()}.png`);
```

Comment:
Do we need to check the colors here, or could that be handled by the `color` test? (see my other comment re: the `color` test)

Also, can we take a screenshot of the individual checkbox?